processing/src/main/java/org/apache/carbondata/processing/store/TablePage.java

```diff
@@ -222,8 +222,9 @@ private void addComplexColumn(int index, int rowId, byte[] complexColumns) {
       ByteBuffer byteArrayInput = ByteBuffer.wrap(complexColumns);
       ByteArrayOutputStream byteArrayOutput = new ByteArrayOutputStream();
       DataOutputStream dataOutputStream = new DataOutputStream(byteArrayOutput);
-      complexDataType.parseAndBitPack(byteArrayInput, dataOutputStream,
-          model.getComplexDimensionKeyGenerator());
+      int startOffset = 0;
```

Comment:
What is the use of `startOffset` when not used?

Reply:
Removed